tikv-client/src/main/java/com/pingcap/tikv/meta/TiColumnInfo.java

```diff
@@ -134,6 +142,7 @@ public TiColumnInfo(long id, String name, int offset, DataType type, boolean isP
     this.isPrimaryKey = isPrimaryKey;
     this.originDefaultValue = "1";
     this.defaultValue = "";
+    this.defaultValueBit = null;
```

Comment:
would be better if we could test it in tikv-client later.

Reply:
nice